shop/src/main/java/ro/msg/learning/shop/controller/mappers/Mapper.java

```diff
@@ -0,0 +1,132 @@
+package ro.msg.learning.shop.controller.mappers;
+
+import org.springframework.stereotype.Component;
+import ro.msg.learning.shop.controller.dto.ResponseCreatedProductDTO;
+import ro.msg.learning.shop.controller.dto.ProductCategoryDTO;
+import ro.msg.learning.shop.controller.dto.ProductDTO;
+import ro.msg.learning.shop.controller.dto.SupplierDTO;
+import ro.msg.learning.shop.domain.entity.Product;
+import ro.msg.learning.shop.domain.entity.ProductCategory;
+import ro.msg.learning.shop.domain.entity.Supplier;
+
+import java.util.Collections;
+import java.util.List;
+
+@Component
+public class Mapper {
```

Comment:
Usually we create Mappers for each entity in order to respect the Single Responsability Principle. Please separate the mappers per Entity.